src/plugins/jh-annotations-panel/jh-annotations-panel.js

```diff
@@ -61,11 +75,15 @@ export default class JHAnnotationsPanel extends Component {
           {miradorAnnos}
         </div>
```

Comment:
I hate to be nitpicky, but I think that indentation helps readability. Could you update the indentation of the JSX here?

``` javascript
<ThemeProvider>
  <CssBaseline>
    {
      <CompanionWindow
        title={t('annotations')}
        paperClassName={ns('window-sidebar-annotation-panel')}
        windowId={windowId}
        id={id}
        titleControls={<AnnotationSettings windowId={windowId} />}
      >
        <div className={classes.section}>
          <Typography component="p" variant="subtitle2">{t('showingNumAnnotations', { number: annotationCount })}</Typography>
        </div>

        <div>
          {miradorAnnos}
        </div>

        <Box pl="8px" pr="8px">
          <div>
            {annoPages}
          </div>
        </Box>

      </CompanionWindow>
    }
  </CssBaseline>
</ThemeProvider>
```

Reply:
I'm not seeing any indentation differences?

Reply:
Comment above comapred to: 
https://github.com/jhu-digital-manuscripts/rosademo-frontend/blob/e36299c898132f625c01ddbe4fa4ae2b4fbde3ee/src/plugins/jh-annotations-panel/jh-annotations-panel.js#L61-L86

The leading and trailing curly brackets should be on separate lines and the `<CompanionWindow>` component and children should be indented

Reply:
Ran a format, pushed change